### PR TITLE
Add option to load app_definition by app_id

### PR DESCRIPTION
**Pull request by maaktweluit**

### CODE REVIEW

File: golem/apps/default.py [hunk 1]
```diff
@@ -45,3 +45,10 @@ def save_built_in_app_definitions(path: Path) -> List[AppId]:
             app_ids.append(app_id)
 
     return app_ids
+
+
+def get_built_in_app_by_id(app_id: AppId) -> Optional[AppDefinition]:
```
Comment (Wiezzel) [inline golem/apps/default.py]: Better make `APPS` a dictionary so we can use simple `get()`.

File: scripts/task_api_tests/basic_integration.py [hunk 2]
```diff
@@ -27,19 +30,21 @@
...
-    app_definition = load_app_from_json_file(Path(app_definition_path))
     env_prerequisites = app_definition.requestor_prereq
     app_manager = appmanager.AppManager()
     app_manager.register_app(app_definition)
     app_manager.set_enabled(app_definition.id, True)
 
     env_manager = envmanager.EnvironmentManager()
-    register_built_in_repositories()
+    docker_repo = repository_from_image_name(
+        app_definition.requestor_prereq['image'])
```
Comment (Wiezzel) [inline scripts/task_api_tests/basic_integration.py]: That's fine for now but it's a hack and would require some action once we add other environments.  Please, put a comment here to remember about it.

File: golem/apps/default.py (hunk 1 above)
Comment (maaktweluit) [reply golem/apps/default.py]: good one, updated!

File: scripts/task_api_tests/basic_integration.py (hunk 2 above)
Comment (maaktweluit) [reply scripts/task_api_tests/basic_integration.py]: Added

**Review by Wiezzel — APPROVED** [review]

**Review by mfranciszkiewicz — APPROVED** [review]

**Review by shadeofblue — APPROVED** [review]
lgtm

**Pull request merged**